pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.3.5</version>
+    <version>3.0.0</version>
```

Comment:
If houskeeping is fine for Jenkins with `hotels-oss-parent` 4.0.0, then I'll update this to v4 as well.